server/src/entity/MenuItem.ts

```diff
@@ -132,4 +131,10 @@ export class MenuItem extends BaseEntity {
     (inventoryitem) => inventoryitem.menuItems
   )
   public ingredients: InventoryItem[];
+
+  @ManyToMany((type) => Order, (order) => order.menuItems)
+  public orders: Order[];
+
+  @OneToMany((type) => Review, (review) => review.menuItem)
```

Comment:
Inconsistency with relationships defined in Order.ts:
```
  @OneToMany((type) => Shift, (shift) => shift.orders)
  public shift: Shift;
```
I think technically your MenuItem.ts is the correct style according to http://typeorm.io/#/many-to-one-one-to-many-relations however, so we may need to ensure the other entities don't have this issue replicated if it matters.